src/spotted/handlers/reply.py

```diff
@@ -16,7 +16,7 @@ async def reply_cmd(update: Update, context: CallbackContext):
     """
     info = EventInfo.from_message(update, context)
 
-    if len(info.text) <= 7:  # the reply is empty
+    if len(info.text) <= 7 or Config.post_get("channel_tag")[1:] in info.text.lower():  # the reply is empty
```

Comment:
but if I have
`/reply@Spotted_DMI_Bot blablabla`
this condition will be true but I am using the reply correctly

Reply:
I think when you use the full-command `/reply@Spotted_DMI_Bot answer` it is only because you accidentally click the suggested command of the bot menu. Usually you just use `/reply answer` without including `@Spotted_DMI_bot` in the command

Reply:
but it's wrong conceptually that you can't use the full-command
you could change the condition into something like
`if len(...) <= SOMETHING:`

where SOMETHING is
- `7` when `not bot_tag in info.text.lower()` 
-  `7 + len(config bot_tag)` when `bot_tag in info.text.lower()`